### ISSUE TITLE
Suggestion: NPC tagged monster

### ISSUE THREAD

**Issue by alizan** [runelite/runelite#13455]
I have a suggestion that would be very useful for ironmen. Let's say you are at brutal black dragons and see someone attack one of the black dragons, it would then be marked orange i.e or a customable color until it is dead or has respawned. It happens too often that NPC is tagged and you dont get loot because someone auto-retalite or leaves mid kill. To highlight these NPC's would be beneficial

If you want to take it a step further, you could calculate time it needs to regen the HP to full so the tag goes away. If this feature is inline with Jagex third party boundaries, it would be extremely useful for ironmen. 
Too many times has the message "As an Iron Man, you don't get loot if other players helped you kill the monster" been displayed

**Comment by hex-agon** [comment]
Sadly this is not fully possible to solve as the client doesn't know any information about a NPC's health until you start attacking it OR you're in range when the npc was attacked. For example, you were returning from the bank and go downstairs, there's no way to know which brutal black dragons were tagged and which were not.

**Comment by alizan** [comment]
That's true unfortunately. However if you are within range of the NPC and see its health go down when someone else attacks it, would it not be possible to highlight it in this case?

**Comment by hex-agon** [comment]
Like i mentioned, yes, that would be the only possible case. The thing is it can become a confusing behavior for anyone that is not aware of this limitation, i think this would be fine for a hub plugin and not a core.

**Comment by Nightfirecat** [comment]
I think we could add this to the existing NPC Indicators plugin. We have a similar type of feature in "Show only own loot" in the Ground Items plugin after all.

**Comment by alizan** [comment]
That would very much appreciated!
